### ISSUE TITLE
Parent meta gets also applied to user meta

### ISSUE THREAD

**Issue by ItsMacs** [lucko/LuckPerms#2762]
### Description
When I set a user to a group and then change their group to another, the user now has the first group's meta set as their personal meta.

### Reproduction steps

1. Run the command /lp user <user> parent set <group>
2. Run the same command but with a different group
3. The user now has the meta of the first group as their personal meta

### Expected behavior
That not to happen.

### Environment details

* Server type/version: `YAPFA-1.15.2 (PaperSpigot)`
* LuckPerms version: `vBukkit 5.2.38`

**Comment by benwoo1110** [comment]
I suggest you send screenshots of `lp user USER info` and `lp user USER meta info`

**Comment by ItsMacs** [comment]
![User info before](https://user-images.githubusercontent.com/70864018/101641723-6576ea80-3a32-11eb-9cbd-e36ef49bb141.png)

![User meta info before](https://user-images.githubusercontent.com/70864018/101641817-82132280-3a32-11eb-8806-eebf7c3a9dd7.png)

![User info after](https://user-images.githubusercontent.com/70864018/101642116-e59d5000-3a32-11eb-97bd-d6588a1ed234.png)

![User meta info after](https://user-images.githubusercontent.com/70864018/101642188-fc43a700-3a32-11eb-92ce-f0924290879a.png)

**Comment by benwoo1110** [comment]
The issue is you have prefix `Manager` set for your own self (not group perms) with a higher weight. You should see https://luckperms.net/wiki/Weight

**Comment by benwoo1110** [comment]
Or maybe what you want to do is also to remove the prefix to set to yourself, since it seems you want the prefix from the group instead. You can use the [command](https://luckperms.net/wiki/Meta-Commands#lp-usergroup-usergroup-meta-removeprefix-priority-prefix-context) or the [web editor](https://luckperms.net/wiki/Web-Editor )

**Comment by ItsMacs** [comment]
The prefix "Manager" was set by the plugin when I set the group "tecnico" to me

**Comment by benwoo1110** [comment]
![image](https://user-images.githubusercontent.com/30431861/101643630-4b0f3200-3a6f-11eb-9a13-5203148f600f.png)

"inherit from self" means you as a player has it, nothing to do with the group.

**Comment by ItsMacs** [comment]
Yeah but when I set myself a group and I run meta clear the group’s prefix gets set as a personal prefix

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by benwoo1110** [comment]
no, clearing prefix wont cause **any** prefix to be set unless you are doing the command wrong. What exact command did you do to "clear the group’s prefix". And note you already have the `Manager` prefix set as a **user** node, so clearing group prefix doesn't nothing to remove that. 

The main issue is you have a prefix set to the **user** itself, and you got confused that it was given due to the first group the player was in, when this not the case.

**Comment by ItsMacs** [comment]
By clearing all the user’s meta with /lp user <user> meta clear, I expect for the meta to be clean, and for the user to only have the group’s prefix.
But, instead, when I do it, LuckPerms automatically sets the group’s prefix to the player as a personal prefix.

I didn’t set the “Manager” tag myself, when I have set the corresponding group to myself, I got that user meta. I didn’t touch any user meta other than cleaning it.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
